src/OrbitGl/GpuTrack.cpp

```diff
@@ -61,8 +62,9 @@ GpuTrack::GpuTrack(CaptureViewElement* parent, TimeGraph* time_graph, orbit_gl::
                                                           capture_data, indentation_level + 1)} {
   timeline_hash_ = timeline_hash;
 
-  std::string timeline =
-      app->GetStringManager()->Get(timeline_hash).value_or(std::to_string(timeline_hash));
+  std::string timeline = orbit_gl::StringManager::GetInstance()
+                             ->Get(timeline_hash)
+                             .value_or(std::to_string(timeline_hash));
```

Comment:
I don't think this is a good idea. If want to remove the dependency to OrbitApp, then `GpuTrack` should get a pointer to a  `StringManager` in its constructor (dependency injection). A singleton will lead to other problems in the future. For example we can't control which part of the code makes mutating changes to StringManager.

Reply:
As discussed offline: This is no longer needed for the current tests and I was wrong about how StringManager is used, so I'm completely removing this commit from the PR. I'm only leaving the `app_ != nullptr` parts required to make the current tests work.